### ISSUE TITLE
1 critical issue for the accessibility test 

### ISSUE THREAD

**Issue by moderia** [moderia/github-issue-template#6]
**Describe the bug**
The accessibility test shows 1 visual issue  in the failing element: <h4 class="h3-mktg mb-3 lh-condensed-ultra text-gradient-purple-coral">

**To Reproduce**
1. Go to https://www.accessibilitychecker.org/
2. paste in https://github.com/moderia/github-issue-template
3. Scroll down to 'Critical Issues'

**Expected behavior**
There should be no critical errors. 

**Screenshots**
https://www.accessibilitychecker.org/audit/?website=https%3A%2F%2Fgithub.com%2Fmoderia%2Fgithub-issue-template&flag=us

**Desktop (please complete the following information):**
 -Google Chrome 

**Smartphone (please complete the following information):**

**Additional context**
